### ISSUE TITLE
Repository is outdated and pulls version 2.3.5

### ISSUE THREAD

**Issue by myopenflixr** [adt-pulse-mqtt/adt-pulse-mqtt#47]
Hello,

I just tried adding this to my Home Assistant and notice that it's pulling 2.3.5 instead of 2.4.

Thanks!

**Comment by ryanmboyce** [comment]
Following this thread as I cannot login with 2.3.5 after fresh install

**Comment by digitalcraig** [comment]
Yes, I did forget to bump the version number in the old repository. The repository which redirects from https://github.com/digitalcraig/hassio only exists for backwards compatibility. I'd recommend making a backup of your add-on configuration and adding the new repository https://github.com/adt-pulse-mqtt/adt-pulse-mqtt instead.

**Comment by myopenflixr** [comment]
I deleted the old repository and added the new one but now I'm getting this error when attempting to install from the new repository
```
Failed to install add-on
The command '/bin/ash -o pipefail -c apk add --update nodejs-npm=~14' returned a non-zero code: 1
```

**Comment by myopenflixr** [comment]
I was comparing the `config.json` files between the two repositories and this line looks to me missing from the `config.json` file in the new repository.

`"image": "adtpulsemqtt/adt-pulse-mqtt-{arch}",`

**Comment by digitalcraig** [comment]
> I was comparing the `config.json` files between the two repositories and this line looks to me missing from the `config.json` file in the new repository.
> 
> `"image": "adtpulsemqtt/adt-pulse-mqtt-{arch}",`

You are correct.. I just fixed that.

**Comment by myopenflixr** [comment]
I "attempted" a pull request for this missing line. It's my first ever pull request so I honestly don't know if I did it correctly.

**Comment by myopenflixr** [comment]
> > I was comparing the `config.json` files between the two repositories and this line looks to me missing from the `config.json` file in the new repository.
> > `"image": "adtpulsemqtt/adt-pulse-mqtt-{arch}",`
> 
> You are correct.. I just fixed that.

I just repulled and can confirm everything is now working perfectly!

Thank you

**Comment by digitalcraig** [comment]
Have you set up 2FA with the new version as well?

**Comment by myopenflixr** [comment]
Yes, 2FA working like a charm!!!!

I have used your repository for the past 1-2 years but switch a couple weeks ago over to another ADT solution in HACS because of the 2FA issue. But that version polled pulse like very 60 seconds which caused motion, doors, and windows sensors to take FOREVER to update.

I came back to your version today and noticed you had implemented 2FA....So I and made the switch back and it's working perfectly!

Great work!

But tomorrow I'm getting upgraded to ADT Command. Sooo...I'll be on the hunt again tomorrow for a new solution to integrate my alarm system.
